vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/HelperTextPageIT.java

```diff
@@ -0,0 +1,39 @@
+package com.vaadin.flow.component.combobox.test;
+
+import com.vaadin.flow.component.combobox.testbench.ComboBoxElement;
+import com.vaadin.flow.testutil.TestPath;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+@TestPath("helper-text")
+public class HelperTextPageIT extends AbstractComboBoxIT {
+
+    @Before
+    public void init() {
+        open();
+    }
+
+    @Test
+    public void assertHelperText() {
+        ComboBoxElement comboHelperText = $(ComboBoxElement.class)
+              .id("combobox-helper-text");
+        Assert.assertEquals("Helper text", comboHelperText.getHelperText());
+
+        clickButton("empty-helper-text");
+        Assert.assertEquals("", comboHelperText.getHelperText());
+    }
+
+    @Test
+    public void assertHelperComponent() {
+        ComboBoxElement comboHelperComponent = $(ComboBoxElement.class)
+              .id("combobox-helper-component");
+        Assert.assertEquals("helper-component",
+              comboHelperComponent.getHelperComponent().getAttribute("id"));
+
+        clickButton("empty-helper-component");
+        Assert.assertEquals("",
+              comboHelperComponent.getHelperComponent().getAttribute("id"));
```

Comment:
I spotted a bug in `ComboBoxElement#getHelperComponent` because of this test. This should return `null` instead of a `TestBenchElement` (on this case it returns the slotted `slot` created by the `combo-box` WC). So I created an issue in TB to address it https://github.com/vaadin/testbench/issues/1269. I believe this PR could be blocked until that is fixed and released (there's an [opened PR](https://github.com/vaadin/testbench/pull/1270) now and hopefully it will not take long).

Reply:
I changed the test, but it will fail for now, of course. Thanks a lot for the fix! :)